ntoskrnl/ke/amd64/mproc.c

```diff
@@ -9,8 +9,10 @@
 
 #include <ntoskrnl.h>
 
+#if 0 // TODO: To enable when implementing below function.
 #define NDEBUG
 #include <debug.h>
+#endif
```

Comment:
Does the UNIMPLEMENTED below spam like crazy or why do you propose that? (I am just asking, because it adds more bloat, but if it would spam aggressively, I could understand). Maybe an UNIMPLEMENTED_ONCE would be a better choice then?

Reply:
Right. Reverted.